migrations/4_alter_test_item_location.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE test_item
```

Comment:
should be up and down files for such updates

Reply:
added